examples/ddpg_gym_tf_pendulum.py

```diff
@@ -0,0 +1,27 @@
+from sandbox.rocky.tf.algos.ddpg.ddpg import DDPG
+from sandbox.rocky.tf.algos.ddpg.noise import OrnsteinUhlenbeckActionNoise
+
+import gym
+import numpy as np
+import tensorflow as tf
+
+RANDOM_SEED = 1234
+
+np.random.seed(RANDOM_SEED)
+tf.set_random_seed(RANDOM_SEED)
```

Comment:
remove these lines, [set_seed(seed)](https://github.com/ryanjulian/rllab/blob/integration/rllab/misc/ext.py#L188) gets called by the user in the launcher which takes care of this